src/cascadia/TerminalSettingsModel/WslDistroGenerator.cpp

```diff
@@ -35,7 +35,7 @@ std::wstring_view WslDistroGenerator::GetNamespace()
 // - <none>
 // Return Value:
 // - a vector with all distros for all the installed WSL distros
-std::vector<Profile> WslDistroGenerator::GenerateProfiles()
+std::vector<Profile> _legacyGenerate()

```

Comment:
Identifiers in the global scope starting with underscore are reserved identifiers.
I think we shouldn't introduce new identifiers like that anymore at this point. `legacyGenerate()`, `namesToProfiles()`, etc. are just fine IMO.